detect_secrets/core/usage.py

```diff
@@ -209,6 +210,19 @@ def _add_adhoc_scanning_argument(self):
                 'plugins\' verdict.'
             ),
         )
+        return self
+
+    def _add_debug_output_raw_argument(self):
+        self.parser.add_argument(
+            '--debug-output-raw',
+            action='store_true',
+            help=(
+                'Outputs the raw secret in the baseline file.'
+                'For development/debug purposes.'
+                'Do NOT use this option in a repo scanning context.'
+            ),
+        )
+        return self
```

Comment:
If we add debug options, we should put this under a different argparse category. e.g. https://github.com/Yelp/detect-secrets/blob/master/detect_secrets/core/usage.py#L336

This way, we can clearly distinguish debug options, and put a large clear warning that tells users not to use it during production runs.